src/modules/header/styles.css

```diff
@@ -0,0 +1,246 @@
+@custom-media --header-breakpoint (min-width: 768px);
+
+:root {
+  --header-menu-width: 12rem;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  line-height: 1; /* FIXME: in another place */
+}
+
+.container li::before {
+  /* What the hell? */
+  content: none;
+}
+
+.nav {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: auto;
+  padding: 0;
+  margin: 0 0 0 auto;
+  transition: .15s;
+
+  @media (--header-breakpoint) {
+    flex-direction: row;
+    width: auto;
+  }
+
+  &:not(.expanded) {
+    height: 0;
+    visibility: hidden;
+
+    @media (--header-breakpoint) {
+      height: auto;
+      visibility: visible;
+    }
+  }
+}
+
+
+.logo-container {
+  margin-right: auto;
+  font-weight: 600;
+}
+
+.logo {
+  height: 1rem;
+  margin-bottom: -1rem;
+}
+
+.toggle-container {
+  display: flex;
+  margin-left: auto;
+
+  @media (--header-breakpoint) {
+    display: none;
+  }
+}
+
+.toggle-button {
+  --icon-color: hsl(var(--component-color) / var(--component-medium-emphasis));
+  padding: .75rem;
+  margin: -1rem;
+  line-height: 1.5;
+  border-radius: 0;
+}
+
+.menu {
+  position: relative;
+  padding: 0;
+  margin: 0;
+}
+
+.menu-title {
+  position: relative;
+  width: 100%;
+  padding-top: var(--app-bar-padding-y, var(--component-padding-y));
+  padding-bottom: var(--app-bar-padding-y, var(--component-padding-y));
+  margin: 0;
+  font-size: 1rem;
+  color: hsl(var(--component-color) / var(--component-medium-emphasis));
+  text-align: left;
+
+  /* Inherited from Material's Subtitle 1 */
+  text-transform: none;
+  letter-spacing: .01em;
+  border-radius: 0;
+
+  @media (--header-breakpoint) {
+    width: auto;
+  }
+
+  &::after {
+    position: absolute;
+    top: var(--app-bar-padding-y, var(--component-padding-y));
+    right: var(--app-bar-padding-x, var(--component-padding-x));
+    display: inline-block;
+    width: 1.5rem;
+    height: 1.5rem;
+    margin: -.25rem;
+    vertical-align: middle;
+    content: '';
+    background:
+      hsl(
+        var(--app-bar-color, var(--component-color)) /
+        var(--app-bar-disabled, var(--component-disabled))
+      );
+    transition: transform 50ms;
+    clip-path: path('M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z');
+
+    @media (--header-breakpoint) {
+      position: static;
+      margin-left: .25rem;
+    }
+  }
+
+  &[aria-expanded='true'] {
+    color: var(--primary);
+
+    &::after {
+      transform: rotate(180deg);
+    }
+  }
+}
+
+.menu-list {
+  display: none;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+
+  &.expanded {
+    display: flex;
+  }
+
+  @media (--header-breakpoint) {
+    position: absolute;
+    top: 100%;
```

Comment:
this rule doesn't change anything, I think you can safely remove it

Reply:
It's safer to have it there. If it's not there, `auto` is set which is equal to regular content placement. It produces a similar result but may get broken if the markup changes.